### ISSUE TITLE
HTMLeditor treat new lines as whitespace

### ISSUE THREAD

**Issue by Integral** [substance/archivist#132]
Just append new line (shift+enter) to some string and save it - you will see whitespace instead of newline.

**Comment by Integral** [comment]
@oliver---- do you have any time today for that? It's most critical now, I think after this we could deploy.

**Comment by michael** [comment]
you think that is critical? most people will just press enter anyway to create a new paragraph?

**Comment by Integral** [comment]
no, the problem that i already create a lot of comments during import and it has a lot of new lines

**Comment by michael** [comment]
can you convert them to paragraphs somehow?

**Comment by michael** [comment]
like instead of:

<p>abc<br>efg</p>

to have

<p>abc</p><p>efg</p>

**Comment by michael** [comment]
it would be replacing `<br>` with `</p><p>`.

**Comment by oliver----** [comment]
I suppose HTMLImporter and Exporter should turn `\n` into `<br>` and vice versa, in general.

**Comment by michael** [comment]
definitely!
